### PR TITLE
FIX: scil_compute_seed_density now uses SFT

### LINKED ISSUE #221: scil_compute_seed_density_map.py bug
@AntoineTheb pense-tu que tu peux tester/checker ce bug.
Les bundles sortent de recobundle scilpy de Francois. Le bundle en question (AF) n'a aucune invalid coordinates. Il visualise bien avec tous les outils...

Data: https://www.dropbox.com/s/b443bzf5z9bmyvj/bug_antoine.zip?dl=0

Merci

scil_compute_seed_density_map.py AF_L_m.trk yo.nii.gz
Traceback (most recent call last):
  File "/Users/desm2239/my_env/scilpy_public/bin/scil_compute_seed_density_map.py", line 4, in <module>
    __import__('pkg_resources').run_script('scilpy==0.1.dev0', 'scil_compute_seed_density_map.py')
  File "/Users/desm2239/my_env/scilpy_public/lib/python3.6/site-packages/pkg_resources/__init__.py", line 658, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/Users/desm2239/my_env/scilpy_public/lib/python3.6/site-packages/pkg_resources/__init__.py", line 1438, in run_script
    exec(code, namespace, namespace)
  File "/Users/desm2239/my_env/scilpy_public/lib/python3.6/site-packages/scilpy-0.1.dev0-py3.6-macosx-10.9-x86_64.egg/EGG-INFO/scripts/scil_compute_seed_density_map.py", line 97, in <module>
    main()
  File "/Users/desm2239/my_env/scilpy_public/lib/python3.6/site-packages/scilpy-0.1.dev0-py3.6-macosx-10.9-x86_64.egg/EGG-INFO/scripts/scil_compute_seed_density_map.py", line 88, in main
    seed_density[tuple(seed_voxel)] += 1
IndexError: index 161 is out of bounds for axis 1 with size 144

### PR DESCRIPTION
**Pull request by AntoineTheb**
Title is self explanatory. Should no longer have problems with different spaces, and fix #221 
See #221 for data

@mdesco @arnaudbore @jchoude

### CODE REVIEW

**Comment by GuillaumeTh** [comment]
@AntoineTheb Thanks ! Your fix works but when you save without --binary the output is still 0 or 1.

**Comment by AntoineTheb** [comment]
@GuillaumeTh I think it's because the data provided only has at most one seed per voxel. Here's a tractogram with more seeds: https://drive.google.com/file/d/13BSfSqEQ4VOZB6s5el1B10lL2gevcGWM/view?usp=sharing

**Comment by GuillaumeTh** [comment]
@AntoineTheb True that ! I was tired I think. LGTM !

**Comment by GuillaumeTh** [comment]
@mdesco Is it ok on your side ?

File: scripts/scil_compute_seed_density_map.py [hunk 1]
```diff
@@ -60,26 +54,22 @@ def main():
...
-    tracts_file = load(args.tractogram_filename, args.lazy_load)
-    if 'seeds' in tracts_file.tractogram.data_per_streamline:
-        seeds = tracts_file.tractogram.data_per_streamline['seeds']
+    # Load files and data. TRKs can have 'same' as reference
+    # Can handle streamlines outside of bbox
+    tractogram = load_tractogram(args.tractogram_filename, 'same',
+                                 bbox_valid_check=False)
+    # Streamlines are saved in RASMM but seeds are saved in VOX
+    # This might produce weird behavior with non-iso
+    tractogram.to_vox()
```
Comment (EmmaRenauld) [inline scripts/scil_compute_seed_density_map.py:63]: In other scripts, we often call the tractograms 'sft'. Helps with clarity I think but just a suggestion.

Comment (EmmaRenauld) [inline scripts/scil_compute_seed_density_map.py:63]: Here you probably should add to_corner() and make sure that line 74 (`seed_voxel = np.round(seed).astype(np.int)`) fits.

**Comment by AntoineTheb** [comment]
@EmmaRenauld here's a tractogram with many seeds per voxel:
https://drive.google.com/file/d/13BSfSqEQ4VOZB6s5el1B10lL2gevcGWM/view?usp=sharing

File: scripts/scil_compute_seed_density_map.py (hunk 1 above)
Comment (AntoineTheb) [reply scripts/scil_compute_seed_density_map.py]: Fixed !

Comment (AntoineTheb) [reply scripts/scil_compute_seed_density_map.py]: Fixed !

Comment (GuillaumeTh) [reply scripts/scil_compute_seed_density_map.py]: @EmmaRenauld Why to corner ?

Comment (GuillaumeTh) [reply scripts/scil_compute_seed_density_map.py]: I double checked with @frheault. It is ok !

**Pull request merged**